### DISCUSSION TITLE
Release 1.1.4 broken

### DISCUSSION THREAD

**Discussion by joelmarty** [quarkiverse/quarkus-github-app#208] [Q&A] [answered]
Hi, it looks like the new release 1.1.4 introduces a bug that prevents the app from starting with the following error:

```
2021-10-15 14:22:19,037 ERROR [io.qua.dep.dev.IsolatedDevModeMain] (main) Failed to start quarkus: java.lang.RuntimeException: io.quarkus.builder.BuildException: Build failure: Build failed due to errors
        [error]: Build step io.quarkiverse.githubapp.deployment.GitHubAppProcessor#replayUi threw an exception: java.lang.NoSuchMethodError: 'io.quarkus.maven.dependency.ResolvedDependency io.quarkus.deployment.util.WebJarUtil.getAppArtifact(io.quarkus.deployment.pkg.builditem.CurateOutcomeBuildItem, java.lang.String, java.lang.String)'
```

This looks like a backwards incompatible change in some supporting library or some library version mismatch.

**Comment by joelmarty** [comment]
Nevermind, the issue was because the 1.1.4 release requires quarkus 2.4.0.CR1 🤦🏻

**Comment by gsmet** [comment, marked as answer]
Yes, I had to upgrade to fix our CI with Quarkus main. I didn't think I would release it but I thought it would be nice for you anyway. Quarkus 2.4.0.Final will come soon.
